src/views/Home/Home.js

```diff
@@ -72,12 +72,15 @@ const Home = () => {
           </span>
         )}
         <br />
-        Every two weeks, the preferred project of the community will receive the
-        interest of{' '}
+        The previous winning project has earned{' '}
```

Comment:
This copy was just wrong, the amount shown is funds going to the previous winner.

There is actually a problem with the contracts here @moose-code which means it will be difficult for us to use the redirect function. See this card https://www.notion.so/daocare/9bfbaba890164e27ba5c3b2407108d49?v=7b67c5b1534f42089ea42a824406e32f&p=8e6e071be09d46739cf06130ac2b9e29 and the relevant code in the aave contracts. https://github.com/aave/aave-protocol/blob/master/contracts/tokenization/AToken.sol#L522 . So I'm strongly recommending to not redirect the interest stream, and give the user DAI (not aDai) at the end of the period when the next winner is selected.

Reply:
My opinion regarding copy is rmove any wording of what the prized will actually be at this point. Until our pool is big enough, we shouldn't be showing any value except the total size of the pool.

Someone seeing our contracts are live and contain 10 000 DAI is far more impressive and compelling than seeing they could win $10 every-week or so. I think we only start bringing these other stats into the equation when the pool is bigger than 50 000 DAI at least.

Agree with you for the long run that redirect interest stream function is maybe not optimal, especially for obtaining the values of interest earned easily for front end (and this is something that would be nice to emit in the event when a winner wins). 

Also, implementing it in a different manner, means we could implement the more optimal veto function, where the user doesn't have to rage quit, but rather, calls a function that for that week redirects the interest stream to themselves if they don't agree with the winner, while still keeping them into the pool. Ideally we don't want users to exit the pool, so this is way more optimal.

Reply:
Will also be easier to implement our business logic of us receiving a small cut to fund dev.

Reply:
I responded on Notion @joaoaguiam :+1: